src/main.rs

```diff
@@ -300,6 +300,7 @@ fn run() -> Result<()> {
         runner.execute(Step::HomeManager, "home-manager", || unix::run_home_manager(&ctx))?;
         runner.execute(Step::Asdf, "asdf", || unix::run_asdf(&ctx))?;
         runner.execute(Step::Mise, "mise", || unix::run_mise(&ctx))?;
+        runner.execute(Step::Aqua, "aqua", || unix::run_aqua(&ctx))?;
```

Comment:
Why limit this to UNIX, looks like Windows is also supported: https://aquaproj.github.io/docs/install

Reply:
I've moved it to generic.rs